docs/sources/variables/templates-and-variables.md

```diff
@@ -0,0 +1,249 @@
+the dashboard. These dropdowns make it easy to change the data being displayed in your dashboard.
+
+{{< docs-imagebox img="/img/docs/v50/variables_dashboard.png" >}}
+
+These can be especially useful for administrators who want to allow Grafana viewers to quickly adjust visualizations but do not want to give them full editing permissions. Grafana Viewers can use variables.
+
+Variables and templates also allow you to single-source dashboards. If you have multiple identical data sources or servers, you can make one dashboard and use variables to change what you are viewing. This simplifies maintenance and upkeep enormously.
+
+## Templates
+
+A _template_ is any query that contains a variable.
+
+For example, if you were administering a dashboard to monitor several servers, you _could_ make a dashboard for each server. Or you could create one dashboard and use panels with template queries like this one:
+
+```
+wmi_system_threads{instance=~"$server"}
+```
+
+Variable values are always synced to the URL using the syntax `var-<varname>=value`.
+
+## Variable syntax
+
+Panel titles and metric queries can refer to variables using two different syntaxes:
+
+- `$varname`
+  This syntax is easy to read, but it does not allow you to use a variable in the middle of a word.
+  **Example:** apps.frontend.$server.requests.count
+- `[[varname]]` 
+  **Note:** This syntax might be deprecated in the future.
+  Example: apps.frontend.[[server]].requests.count
+
+Before queries are sent to your data source the query is _interpolated_, meaning the variable is replaced with its current value. During
+interpolation the variable value might be _escaped_ in order to conform to the syntax of the query language and where it is used.
+For example, a variable used in a regex expression in an InfluxDB or Prometheus query will be regex escaped. Read the data source specific
+documentation topic for details on value escaping during interpolation.
+
+For advanced syntax to override data source default formatting, refer to [Advanced variable format options]({{< relref "advanced-variable-format-options.md" >}}).
+
+## Examples of templates and variables
+
+To see variable and template examples, go to any of the dashboards listed below.
+
+- [Elasticsearch Templated dashboard](https://play.grafana.org/dashboard/db/elasticsearch-templated)
+- [Graphite Templated Nested dashboard](https://play.grafana.org/dashboard/db/graphite-templated-nested)
+- [InfluxDB Templated dashboard](https://play.grafana.org/dashboard/db/influxdb-templated)
+
+Variables are listed in dropdown lists across the top of the screen. Select different variables to see how the visualizations change. 
+
+To see variable settings, navigate to **Dashboard Settings > Variables**. Click a variable in the list to see its settings.
+
+Templates are in the query portion of panels. Queries with text that starts with `$` are templates. Not all panels will have template queries.
```

Comment:
I would express this differently. Using the word `templates` like this is confusing for me. It is such a generic word that I wasn't sure exactly what you meant. Maybe `templated queries` would be clearer? It is still a query even if it is also a template.

The whole sentence does not add anything for me:  `Templated queries are in the query portion of panels`.